packages/sdk/tasks/plugin/retire.ts

```diff
@@ -0,0 +1,73 @@
+import { constants } from "ethers";
+import { task } from "hardhat/config";
+
+import { CErc20PluginDelegate } from "../../typechain/CErc20PluginDelegate";
+import { FuseFeeDistributor } from "../../typechain/FuseFeeDistributor";
+import { MidasERC4626 } from "../../typechain/MidasERC4626";
+
+task("plugins:beefy:retire", "Retires a Beefy plugin that is marked as EOL").setAction(
```

Comment:
Would suggest refactoring this to take in the market address so we can use it in the future, but that's not a high prio rn